lib/puppet/provider/truststore/keytool.rb

```diff
@@ -0,0 +1,13 @@
+Puppet::Type.type(:truststore).provide(:keytool) do
```

Comment:
```suggestion
require 'puppet_x/certs/keystore'

Puppet::Type.type(:truststore).provide(:keytool) do
```

Reply:
I believe this was missed

Reply:
Thanks, I will address after the underlying (the keystore) PR is merged.